fri/src/two_adic_pcs.rs

```diff
@@ -481,7 +484,7 @@ fn compute_inverse_denominators<F: TwoAdicField, EF: ExtensionField<F>, M: Matri
                 batch_multiplicative_inverse(
                     &subgroup[..(1 << log_height)]
                         .iter()
-                        .map(|&x| EF::from_base(x) - z)
+                        .map(|&x| z - x)
```

Comment:
While I agree with this change, this does negate the output of this function. 

Can you add a doc comment to the function describing what it does/outputs and also update the comment on line 265 of the file (immediately above the declaration of `inv_denoms`) from `X - z` to `z - X`.